### PR TITLE
Chore/onboarding client

### PR DESCRIPTION
**Pull request by matthewcarlreetz**
Use new helium-js onboarding client
Move onboarding txn post out of the sdk

### CODE REVIEW

File: example/src/AssertLocation/AssertLocation.tsx [hunk 1]
```diff
@@ -67,8 +68,10 @@ const AssertLocation = () => {
     getHotspotDetails(gatewayAddress)
       .then(setHotspot)
       .catch((e) => console.log(e))
-    Onboarding.getOnboardingRecord(gatewayAddress)
-      .then(setOnboardingRecord)
+
+    new OnboardingClient()
```
Comment (tyler-whitman) [inline example/src/AssertLocation/AssertLocation.tsx:72]: Would it be better to create a global instance somewhere? Or is it intended to make a new instance each time we interact with the `OnboardingClient `?

File: src/HotspotBle/useHotspotBle.tsx [hunk 2]
```diff
@@ -41,6 +41,7 @@ const useHotspotBle = () => {
   const instanceRef = useRef<BleManager | null>(null)
   const [devices, setDevices] = useState<Record<string, Device>>({})
   const [device, setDevice] = useState<Device>()
+  const [onboardingClient] = useState(new OnboardingClient())
```
Comment (tyler-whitman) [inline src/HotspotBle/useHotspotBle.tsx:44]: Why does it need to be a part of the state?

**Review by tyler-whitman — APPROVED** [review]

File: example/src/AssertLocation/AssertLocation.tsx (hunk 1 above)
Comment (matthewcarlreetz) [reply example/src/AssertLocation/AssertLocation.tsx:72]: Ideally makers would create a global instance, but for the sake of a tiny demo I think this is fine

File: src/HotspotBle/useHotspotBle.tsx (hunk 2 above)
Comment (matthewcarlreetz) [reply src/HotspotBle/useHotspotBle.tsx:44]: It doesn't, that's just one way of keeping a stable instance. That being said I shouldn't have this here at all. This needs to be moved out of the sdk, so the makers can configure their own global instance.

**Pull request merged**